### PR TITLE
feat: auto clean obsoleted package after install new ones

### LINKED ISSUE #169: Automatically uninstall deprecated package
When upgrading new package with Quelpa, old package should be uninstalled automatically

### PR DESCRIPTION
**Pull request by kiennq**
This should fix #169 
Now, old packages will be automatically removed when newer one is installed.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -136,7 +136,8 @@ To run =quelpa-upgrade-all= at most every 7 days, after all the init files are l
 
 Quelpa installs packages using Emacs's built-in package library, =package.el=, so after installing a package with Quelpa, you can view its status and remove it using =M-x list-packages RET=.  Note that deleting a package this way does not yet affect Quelpa's cache, so Quelpa will still consider the package to have been installed with Quelpa.
 
-As well, Quelpa does not automatically remove obsolete package versions after upgrading.  To delete obsolete package versions manually, use =M-x list-packages RET=, then press =~= to mark all obsolete packages for deletion, and press =x= to delete them.
++As well, Quelpa does not automatically remove obsolete package versions after upgrading.  To delete obsolete package versions manually, use =M-x list-packages RET=, then press =~= to mark all obsolete packages for deletion, and press =x= to delete them.+
+Quelpa will automatically remove obsoleted package versions after upgrading.
```
Comment (steckerhalter) [inline README.org]: hmm, so quelpa does both remove them and not remove them :question: 
I guess the manual instruction can be removed if quelpa does it automatically

Comment (kiennq) [reply README.org]: The part of not automatically remove is displayed as being crossed through. I intentionally do that so user will notice the changes

Comment (tsuu32) [reply README.org]: This statements confuse me (maybe other users). To notice the changes I think we should write about the changes in NEWS section.

Comment (kiennq) [reply README.org]: Ok, I update the doc with #174 
Let me know if that's good enough.

Comment (steckerhalter) [reply README.org]: yeah I think to address this change in the NEWS section would make more sense. @kiennq could you add this to #174 ?

Comment (kiennq) [reply README.org]: Added, let me know if I should change more

**Pull request merged**